src/OrbitDB.js

```diff
@@ -385,6 +387,40 @@ class OrbitDB {
     logger.debug(`Saved manifest to IPFS as '${dbAddress.root}'`)
   }
 
+  async _attemptMigration(options, dbAddress) {
+    let oldCache = this.caches[options.directory], oldStore
```

Comment:
I'm confused of the `,` after the assignment. How about declaring `oldStore` explicitly and putting on its own line?